netcode.c

```diff
@@ -4019,6 +4041,89 @@ int netcode_server_find_free_client_index( struct netcode_server_t * server )
     return -1;
 }
 
+/**
+ * @brief skillz_add_client_to_match
+ * @param server
+ * @param skillz_match_id
+ * @param client_id
+ * @return If client was successfully added to a match or a match was created return 1.  Else return 0.
+ *
+ */
+int skillz_add_client_to_match(struct netcode_server_t * server, int skillz_match_id, uint64_t client_id)
+{
+    skillz_match_t * m;
```

Comment:
It might make things quite verbose, but I think it might be better to avoid 1 letter variables and call this `match` instead.